screens/explorer/maps/map-minimap.module.css

```
@@ -0,0 +1,81 @@
+  display: flex;
+  align-items: center;
+  gap: 2px;
+  /*
+   * Markers scale with the width of the map instead of being a fixed size: a 4v4 map carries over 30
+   * points, and at phone widths fixed size markers cover each other. The upper bound is the size we
+   * want on a desktop, the lower one keeps them tappable. The plain value is the fallback for
+   * browsers without container query units.
+   */
+  /* `--point-icon-scale` is set per marker from the tier of its point, see `MP_MAP_POINT_TIER_SCALE`. */
+  --point-icon-size: 28px;
+  --point-icon-size: calc(clamp(20px, 4.3cqw, 28px) * var(--point-icon-scale, 1));
```

Stylelint (17.14.1)

[error] Duplicate custom property "--point-icon-size" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)